composer.json

```diff
@@ -49,7 +50,16 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "cweagans/composer-patches": true
+        }
+    },
+    "extra": {
+        "enable-patching": true,
+        "patches": {
+            "drupal/webform_rest": {
+                "Added ability to modify response data sent from Webform Submission endpoint": "https://raw.githubusercontent.com/itk-dev/os2forms_rest_api/feature/linked-data/patches/webform_rest_submission.patch"
+            }
         }
```

Comment:
I suggest to keep patch path as a local path, not github URL. That helps to avoid problem if this is changed or is not available anymore for any reasons

Reply:
I would prefer this as well, but we cannot use a local path (relative to the module path) to the patch due to [a limitation in composer-patches](https://github.com/cweagans/composer-patches/issues/339#issuecomment-729841118).

As noted in the [pull request description](https://github.com/OS2Forms/os2forms_rest_api/pull/2#issue-1601341772) we need to update the url to something that we can rely on when merged, e.g. `https://raw.githubusercontent.com/OS2Forms/os2forms_rest_api/1.1.0/patches/webform_rest_submission.patch`. We could also use a gist, but keeping the patch with the module seems the best solution of the two.